packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -37,8 +44,26 @@ const propTypes = {
 
 class EmbeddedContentConsumer extends React.Component {
   componentDidMount() {
+    // Merging the iframe options props
+    const frameOptions = { ...this.props.options };
+
+    if (this.props.title) {
+      if (frameOptions.iframeAttrs === null
+        || frameOptions.iframeAttrs === undefined) {
+        frameOptions.iframeAttrs = {};
+      }
+
+      Object.assign(frameOptions.iframeAttrs, { title: this.props.title });
```

Comment:
Similar comment about why we're doing this deleting of untruthy keys: Is this really necessary?

Reply:
My initial thought was that we don't want any key without a corresponding value set. But, I could see consumers wanting to have some key set specifically with null or undefined.

I can remove that check.

Reply:
I removed that check to allow more flexibility for the consumers.
https://github.com/cerner/terra-framework/pull/1776/commits/3ffd8305ee593960880ba6b3ac9bbdab84932e6c